palantirnet/snowflake
# 7 evaluator

Adds an `evaluator` field to the form and URL. Also fixes a table issue.

To test:

* Checkout this branch
* Run `yarn dev`
* Go to http://localhost:3000/#5,5,5,5,5,5,5,5,5,5,5,5,5,1,Palantiri%20Name,Project%20Management,Evaluator%20Name/Team
* Confirm that you can edit the `Evaluator` field.
* Confirm that your edits are reflected in the URL
* Confirm that only 15 columns are visible in the footer table.
